node/src/types/chainspec/accounts_config.rs

```diff
@@ -95,37 +95,142 @@ impl FromBytes for AccountConfig {
     }
 }
 
+#[derive(PartialEq, Eq, Serialize, Deserialize, DataSize, Debug, Copy, Clone)]
+pub struct DelegatorConfig {
+    validator_public_key: PublicKey,
+    delegator_public_key: PublicKey,
+    balance: Motes,
+    delegated_amount: Motes,
+}
+
+impl DelegatorConfig {
+    pub fn new(
+        validator_public_key: PublicKey,
+        delegator_public_key: PublicKey,
+        balance: Motes,
+        delegated_amount: Motes,
+    ) -> Self {
+        Self {
+            validator_public_key,
+            delegator_public_key,
+            balance,
+            delegated_amount,
+        }
+    }
+
+    pub fn validator_public_key(&self) -> PublicKey {
+        self.validator_public_key
+    }
+
+    pub fn delegator_public_key(&self) -> PublicKey {
+        self.delegator_public_key
+    }
+
+    pub fn balance(&self) -> Motes {
+        self.balance
+    }
+
+    pub fn delegated_amount(&self) -> Motes {
+        self.delegated_amount
+    }
+}
+
+impl Distribution<DelegatorConfig> for Standard {
```

Comment:
We've not generally been implementing `Distribution` for types in casper-node, but instead preferring a `random()` method [like this one](https://github.com/CasperLabs/casper-node/blob/master/node/src/types/chainspec/network_config.rs#L44-L59).

If you disagree, then at least this could be feature-gated so it's not available outside of tests?

Reply:
did both, for good measure